sdk/kusto/Azure.ResourceManager.Kusto/samples/Generated/Samples/Sample_KustoDatabaseResource.cs

```diff
@@ -407,18 +407,18 @@ public async Task ValidateDataConnection_KustoDataConnectionEventGridValidation(
                     DatabaseRouting = KustoDatabaseRouting.Single,
                 },
             };
-            ArmOperation<DataConnectionValidationResults> lro = await kustoDatabase.ValidateDataConnectionAsync(WaitUntil.Completed, content);
+            ArmOperation<DataConnectionValidationResults> lro = await kustoDatabase.DataConnectionValidationDataConnectionAsync(WaitUntil.Completed, content);
```

Comment:
mapping failed: https://github.com/Azure/azure-sdk-for-net/blob/91af14e56d8b9f12d66fc5217e2e13670a21d8e1/sdk/kusto/Azure.ResourceManager.Kusto/src/autorest.md?plain=1#L31